docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md

```diff
@@ -0,0 +1,87 @@
+# 🌟 Couchers.org Volunteer Newsletter
+
+## 🎉 Welcome to Our First Edition!
+
+Hello, amazing Couchers volunteers!  
+If you’re reading this, you’ve volunteered with Couchers in some capacity. With the New Year underway, we have some resolutions of our own and big goals going forward. We’ve had a surge of volunteer activity, so we wanted to share with you what we’ve been working on, what we plan to do, and what we could use help with in case this stirs any FOMO 😉.
+
+[[[[ PHOTO OF VOLUNTEERS CALL - SOMEONE SEND IT TO ME PLZZZ ]]]]
+
+---
+
+## 💻 Engineering Updates
+
+Here’s what we’ve been working on:
+
+* 👤 **Complete Profiles**: Incomplete profiles can now be filtered out on the map. Users with incomplete profiles won’t be able to send messages, host requests, or create events until they upload a photo and fill in their details.
+* 🛡️ **Strong Verification**: Users can now verify gender and age using their passports for enhanced safety. Shoutout to [Aapeli](https://github.com/aapeliv) for setting this up and upgrading it recently!
+* 🔔 **Push Notifications**: Notifications are in Feature Preview, with a notification settings page complete. Backend work is mostly done, and frontend updates are in progress. Care to help? [View the issue here](https://github.com/Couchers-org/couchers/issues/5006).
+* 🗺️ **More Map Filters**: Filters for gender, meetup status, and excluding empty profiles are live! Shoutout to [David](https://github.com/bakeiro) for all his work on the map!
+* 🎨 **Events Page Revamp**: The events page has been totally redesigned! Users can now cancel events as well, with more improvements planned for the future.
+* 📱 **Mobile Improvements**:
+    * **Standalone Mobile App**: Shoutout to [Ivan](https://github.com/polemius), who has been working solo on this!
+    * **Mobile Web Enhancements**: Progress continues (though we still need more hands).
+* ⚙️ **Frontend Upgrades**: We’ve upgraded React to v18, Material-UI, and Next.js to v13. Shoutout to [Nicole](https://github.com/nabramow), your author, for this!
+
+---
+
+## 🚀 What’s Next for Engineering & How You Can Help
+
+We’d love your help with:
+
+* 📱 **React Native Mobile App**: We could use more hands here. Help [Ivan](https://github.com/polemius) tackle the standalone app!
+* 🗺️ **Map Redesign**: Design and development support needed! Check out the [map redesign ticket](https://github.com/Couchers-org/couchers/issues).
+* 🌐 **Public Events**: We’re adding a feature to create shorter, public versions of events so they can be shared with non-Couchers. [More info in the epic.](https://github.com/Couchers-org/couchers/issues/5339)
+* ⬆️ **Frontend Package Upgrades**: Help us stay current by upgrading to React v19, Next.js v14, Material-UI v6, and react-query v3, among others.
+* 🚫 **User Blocking & Reporting**: We’re working on implementing user-blocking functionality, expanding reporting to comments and implementation of our private feedback flow (designs ready)
+
```

Comment:
Is there a ticket for this? I'd love to link it but can't find anything @jesseallhands

Reply:
Added it!